apps/arena/lib/arena/entities.ex

```diff
@@ -46,9 +50,11 @@ defmodule Arena.Entities do
       is_moving: false,
       aditional_info: %{
         team: team,
-        health: character.base_health,
-        base_health: character.base_health,
-        max_health: character.base_health,
+        health: (character.base_health + character.base_health * @stat_increase_per_level * (level - 1)) |> round(),
+        base_health:
+          (character.base_health + character.base_health * @stat_increase_per_level * (level - 1)) |> round(),
+        max_health: (character.base_health + character.base_health * @stat_increase_per_level * (level - 1)) |> round(),
+        base_attack: character.base_attack + character.base_attack * @stat_increase_per_level * (level - 1),
```

Comment:
Create a function to do this instead of doing it 4 times.

Reply:
done in [953d11f](https://github.com/lambdaclass/mirra_backend/pull/1165/commits/953d11f226c19c113e51ac4463f34b196326fe29)